labgrid/driver/qemudriver.py

```diff
@@ -192,6 +204,9 @@ def get_qemu_base_args(self):
             cmd.append("-bios")
             cmd.append(
                 self.target.env.config.get_image_path(self.bios))
+        elif self._bios_fname:
+            cmd.append("-bios")
+            cmd.append(self._bios_fname)
```

Comment:
Now we can provide `bios` via the client arguments and than set `_bios_fname` via the `set_bios()` function. Why don't you modify `bios` instead?

Reply:
Well in this case 'bios' would not be provided.

The point here is that the strategy can provide the firmware, no matter what board type is chosen. We don't have to have a different 'bios' for every architecture / target.

Reply:
In case this driver is used by your strategy it is not provided, but it is still possible. Please modify this to use `self.bios` instead of the new hidden `self._bios_fname`.